Test placeholder attribute in the textarea element

haoxli opened the pull request.

Commit:
Test placeholder attribute in the textarea element

Ms2ger commented on html/semantics/forms/the-textarea-element/textarea-placeholder-manaul.html:
```diff
@@ -0,0 +1,12 @@
+<meta charset="utf-8">
+<title>HTML Test: textarea - placeholder attribute</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://www.w3.org/TR/html5/forms.html#the-textarea-element">
```
We point to the most recent spec: <https://html.spec.whatwg.org/multipage/#attr-textarea-placeholder>.

Ms2ger commented:
s/manaul/manual/ in the test name.

Commit:
Update help link and test name

- point to the most recent spec: https://html.spec.whatwg.org/multipage/#attr-textarea-placeholder
- s/manaul/manual/ in the test name.

Commit:
Update assert description more clearly
